core/unit_test/TestReducers.hpp

```diff
@@ -1012,11 +1221,19 @@ struct TestReducers {
 #if !defined(KOKKOS_ENABLE_OPENACC)
     // FIXME_OPENACC - OpenACC (V3.3) does not support custom reductions.
     test_minloc(10003);
+// FIXME_NVHPC misaligned memory
+#if !defined(KOKKOS_COMPILER_NVHPC)
```

Comment:
Since it looks like the OpenMPTarget build is failing and more changes will be needed, I would like you to put the FIXME comment on the same line as the `#ifndef`.  Same comment below.

Reply:
The suggestion was meant for comments that fit on the line